.github/workflows/push-build.yaml

```diff
@@ -0,0 +1,31 @@
+name: Build Image
+on:
+  push:
+    branches:
+      - master
+      - 'releases/**'
+    tags:
+      - v*
+jobs:
+  push-master-image-on-docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@master
+        with:
+          platforms: all
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: build and release
+        env:
```

Comment:
env:
        GIT_PSW: ${{ secrets.GIT_PSW }}
        BRANCH_NAME: ${{ github.base_ref }}

Let's use this instead of `${GITHUB_REF#refs/heads/}`

Reply:
`github.base_ref` is only for `pull_request event` this is not working...show I have to make tweak a little to work